assets/js/modules/analytics/datastore/properties.js

```diff
@@ -378,28 +239,10 @@ export const reducer = ( state, { type, payload } ) => {
 			};
 		}
 
-		case FINISH_FETCH_PROPERTIES_PROFILES: {
-			const { accountID } = payload;
-
-			return {
-				...state,
-				isFetchingPropertiesProfiles: {
-					...state.isFetchingPropertiesProfiles,
-					[ accountID ]: false,
-				},
-			};
-		}
-
-		case CATCH_FETCH_PROPERTIES_PROFILES: {
-			const { accountID, error } = payload;
-
+		case RECEIVE_PROPERTIES_PROFILES_COMPLETION: {
 			return {
 				...state,
-				error,
-				isFetchingPropertiesProfiles: {
-					...state.isFetchingPropertiesProfiles,
-					[ accountID ]: false,
-				},
+				isAwaitingPropertiesProfilesCompletion: false,
```

Comment:
I'm not sure we need this additional flag - see my other comment in the isDoing selector.